diff --git a/.config/hypr/hyprland.conf b/.config/hypr/hyprland.conf
index 252a8d6..a645b8d 100644
--- a/.config/hypr/hyprland.conf
+++ b/.config/hypr/hyprland.conf
@@ -271,22 +271,22 @@ bind = $mainMod CTRL, TAB, movegroupwindow
 bind = $mainMod CTRL SHIFT, TAB, movegroupwindow, b
 
 # Conditional Keymaps:
-bind = $mainMod ALT, E, workspace, e+1
-bind = $mainMod ALT, Y, workspace, e-1
-bind = $mainMod SHIFT, E, movetoworkspace, +1
-bind = $mainMod SHIFT, Y, movetoworkspace, -1
-bind = $mainMod CTRL, Y, resizeactive, -40 0
-bind = $mainMod CTRL, H, resizeactive, 0 40
-bind = $mainMod CTRL, A, resizeactive, 0 -40
-bind = $mainMod CTRL, E, resizeactive, 40 0
-bind = $mainMod SHIFT CTRL, Y, movewindow, l
-bind = $mainMod SHIFT CTRL, H, movewindow, d
-bind = $mainMod SHIFT CTRL, A, movewindow, u
-bind = $mainMod SHIFT CTRL, E, movewindow, r
-bind = $mainMod ALT CTRL, Y, movewindoworgroup, l
-bind = $mainMod ALT CTRL, H, movewindoworgroup, d
-bind = $mainMod ALT CTRL, A, movewindoworgroup, u
-bind = $mainMod ALT CTRL, E, movewindoworgroup, r
+bind = $mainMod ALT, L,  workspace, e+1
+bind = $mainMod ALT, H, workspace, e-1
+bind = $mainMod SHIFT, L,  movetoworkspace, +1
+bind = $mainMod SHIFT, H, movetoworkspace, -1
+bind = $mainMod CTRL, H, resizeactive, -40 0
+bind = $mainMod CTRL, J, resizeactive, 0 40
+bind = $mainMod CTRL, K,  resizeactive, 0 -40
+bind = $mainMod CTRL, L,  resizeactive, 40 0
+bind = $mainMod SHIFT CTRL, H, movewindow, l
+bind = $mainMod SHIFT CTRL, J, movewindow, d
+bind = $mainMod SHIFT CTRL, K,  movewindow, u
+bind = $mainMod SHIFT CTRL, L,  movewindow, r
+bind = $mainMod ALT CTRL, H, movewindoworgroup, l
+bind = $mainMod ALT CTRL, J, movewindoworgroup, d
+bind = $mainMod ALT CTRL, K,  movewindoworgroup, u
+bind = $mainMod ALT CTRL, L,  movewindoworgroup, r
 
 bindel = ,XF86AudioRaiseVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%+
 bindel = ,XF86AudioLowerVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%-
diff --git a/.config/rofi/bluetooth-menu.rasi b/.config/rofi/bluetooth-menu.rasi
index faa90bb..7fc25cd 100644
--- a/.config/rofi/bluetooth-menu.rasi
+++ b/.config/rofi/bluetooth-menu.rasi
@@ -13,11 +13,13 @@ configuration {
     kb-move-end: "";
     kb-move-word-back: "";
     kb-move-word-forward: "";
+    kb-mode-complete: "";
+    kb-remove-to-eol: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y,Ctrl+Left";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e,Ctrl+Right";
+    kb-mode-previous: "H,Ctrl+h,Ctrl+Left";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l,Ctrl+Right";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
diff --git a/.config/rofi/config.rasi b/.config/rofi/config.rasi
index 27fa208..8c6bb51 100644
--- a/.config/rofi/config.rasi
+++ b/.config/rofi/config.rasi
@@ -13,11 +13,13 @@ configuration {
     kb-move-end: "";
     kb-move-word-back: "";
     kb-move-word-forward: "";
+    kb-mode-complete: "";
+    kb-remove-to-eol: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y,Ctrl+Left";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e,Ctrl+Right";
+    kb-mode-previous: "H,Ctrl+h,Ctrl+Left";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l,Ctrl+Right";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
diff --git a/.config/rofi/power-menu.rasi b/.config/rofi/power-menu.rasi
index 22b799c..5377441 100644
--- a/.config/rofi/power-menu.rasi
+++ b/.config/rofi/power-menu.rasi
@@ -13,11 +13,13 @@ configuration {
     kb-move-end: "";
     kb-move-word-back: "";
     kb-move-word-forward: "";
+    kb-mode-complete: "";
+    kb-remove-to-eol: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y,Ctrl+Left";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e,Ctrl+Right";
+    kb-mode-previous: "H,Ctrl+h,Ctrl+Left";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l,Ctrl+Right";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
diff --git a/.config/rofi/wifi-menu.rasi b/.config/rofi/wifi-menu.rasi
index 15f9085..2dcef9b 100644
--- a/.config/rofi/wifi-menu.rasi
+++ b/.config/rofi/wifi-menu.rasi
@@ -13,11 +13,13 @@ configuration {
     kb-move-end: "";
     kb-move-word-back: "";
     kb-move-word-forward: "";
+    kb-mode-complete: "";
+    kb-remove-to-eol: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y,Ctrl+Left";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e,Ctrl+Right";
+    kb-mode-previous: "H,Ctrl+h,Ctrl+Left";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l,Ctrl+Right";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
