diff --git a/lua/ltex-utils/init.lua b/lua/ltex-utils/init.lua
index fdf0381..01ef509 100644
--- a/lua/ltex-utils/init.lua
+++ b/lua/ltex-utils/init.lua
@@ -63,6 +63,9 @@ local function autocmd_ltex()
 		pattern = "TelescopePreviewerLoaded",
 		callback = function(args)
 			---@type string
+            if args.data == nil or args.data.bufname == nil then
+                return
+            end
 			local extension = args.data.bufname:match("%.(%w+)$")
 			if extension == "md" or extension == "tex" then
 				vim.wo.number = Config.rule_ui.previewer_line_number
